From 44375ee2a4ec8950292e7d115fadc88084f84670 Mon Sep 17 00:00:00 2001
From: Joshua Wong Weng Onn <joshua.weng.onn.wong@intel.com>
Date: Thu, 28 Jun 2018 13:36:05 +0800
Subject: [PATCH 084/100] ASoC: Intel: Change be_id variable in machine driver

The be_id variable has been replaced with id. Hence, change be_id to id for the
bxt_gpmrb machine driver.

Signed-off-by: Joshua Wong Weng Onn <joshua.weng.onn.wong@intel.com>
---
 sound/soc/intel/boards/bxt_gpmrb.c | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/sound/soc/intel/boards/bxt_gpmrb.c b/sound/soc/intel/boards/bxt_gpmrb.c
index c89f31a..4e5bbcd 100644
--- a/sound/soc/intel/boards/bxt_gpmrb.c
+++ b/sound/soc/intel/boards/bxt_gpmrb.c
@@ -479,7 +479,7 @@ static int broxton_gpmrb_tdf8532_startup(struct snd_pcm_substream *substream)
 	{
 		/* SSP0 - BT */
 		.name = "SSP0-Codec",
-		.be_id = 1,
+		.id = 1,
 		.cpu_dai_name = "SSP0 Pin",
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
@@ -492,7 +492,7 @@ static int broxton_gpmrb_tdf8532_startup(struct snd_pcm_substream *substream)
 	{
 		/* SSP1 - HDMI-In */
 		.name = "SSP1-Codec",
-		.be_id = 1,
+		.id = 1,
 		.cpu_dai_name = "SSP1 Pin",
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
@@ -503,7 +503,7 @@ static int broxton_gpmrb_tdf8532_startup(struct snd_pcm_substream *substream)
 	{
 		/* SSP2 - Dirana */
 		.name = "SSP2-Codec",
-		.be_id = 1,
+		.id = 1,
 		.cpu_dai_name = "SSP2 Pin",
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
@@ -514,7 +514,7 @@ static int broxton_gpmrb_tdf8532_startup(struct snd_pcm_substream *substream)
 	{
 		/* SSP2-B - Dirana */
 		.name = "SSP2-B-Codec",
-		.be_id = 1,
+		.id = 1,
 		.cpu_dai_name = "SSP2-B Pin",
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
@@ -525,7 +525,7 @@ static int broxton_gpmrb_tdf8532_startup(struct snd_pcm_substream *substream)
 	{
 		/* SSP2-C - Dirana */
 		.name = "SSP2-C-Codec",
-		.be_id = 1,
+		.id = 1,
 		.cpu_dai_name = "SSP2-C Pin",
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
@@ -536,7 +536,7 @@ static int broxton_gpmrb_tdf8532_startup(struct snd_pcm_substream *substream)
 	{
 		/* SSP2-D - Dirana */
 		.name = "SSP2-D-Codec",
-		.be_id = 1,
+		.id = 1,
 		.cpu_dai_name = "SSP2-D Pin",
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
@@ -547,7 +547,7 @@ static int broxton_gpmrb_tdf8532_startup(struct snd_pcm_substream *substream)
 	{
 		/* SSP4 - Amplifier */
 		.name = "SSP4-Codec",
-		.be_id = 0,
+		.id = 0,
 		.cpu_dai_name = "SSP4 Pin",
 		.codec_name = "i2c-INT34C3:00",
 		.codec_dai_name = "tdf8532-hifi",
@@ -558,7 +558,7 @@ static int broxton_gpmrb_tdf8532_startup(struct snd_pcm_substream *substream)
 	{
 		/* SSP5 - TestPin */
 		.name = "SSP5-Codec",
-		.be_id = 1,
+		.id = 1,
 		.cpu_dai_name = "SSP5 Pin",
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
-- 
1.9.1

